src/style.css

```
@@ -0,0 +1,79 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  flex-basis: 33%;
+  flex-grow: 1;
+  color: #fff;
+}
+
+.box-1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.box-2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.box-3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.box-4 {
+  background-color: rgb(40%, 0%, 0%);
+  order: 3;
+}
+
+.box-5 {
+  background-color: rgb(20%, 0%, 0%);
+  order: 2;
+}
+
+.box-6 {
+  order: 1;
+  background-color: rgb(0%, 0%, 0%);
+}
+
+@media (max-width: 899px) {
```

Use `min width` instead
```suggestion
@media (max-width: 899px) {
```